encrypted-config-value-bundle-dropwizard1/src/main/java/com/palantir/config/crypto/EncryptConfigValueCommand.java

```diff
@@ -60,6 +61,6 @@ public void run(Bootstrap<?> bootstrap, Namespace namespace) throws Exception {
         EncryptedValue encryptedValue = algorithm.getEncryptedValue(value, keyWithAlgorithm);
 
         // print the resulting encrypted value to the console
-        System.out.println(encryptedValue);
+        System.out.println(encryptedValue.serialize());
```

Comment:
Would it make sense to define the `toString` to return the serialized representation so that these calls can stay as-is?